components/collator/src/tests.rs

```diff
@@ -469,7 +476,7 @@ fn test_ja_base() {
     let mut options = CollatorOptions::new();
     options.set_strength(Some(Strength::Primary));
 
-    let collator: Collator = Collator::try_new(&data_provider, locale, options).unwrap();
+    let collator: Collator = Collator::try_new(&data_provider, &locale.into(), options).unwrap();
```

Comment:
Suggestion: consider cleaning these up. Instead of
```
let locale: Locale = langid!("ja").into()
...
&locale.into()
```
these could just be
```
&locale!("ja").into()
```

Reply:
I prefer the consistency across most of the tests of using a `let locale = ...` variable.

Reply:
But make it a `DataLocale`? Otherwise you'll get `&(&locale).into()`

Reply:
In almost all places `&locale.into()` works fine. The couple places where I did `&(&locale).into()` I was able to change to `&Default::default()`